Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN groupadd taskbroker --gid 1000 && useradd --gid taskbroker --uid 1000 taskbr
 RUN apt-get update && apt-get upgrade -y && apt-get install -y libssl-dev libsqlite3-dev libsqlite3-0 openssl ca-certificates
 
 EXPOSE 50051
+EXPOSE 50055
```

Comment:
expose both since IIRC the docker image is used for both cloudbuild (prod) and GHCR (devservices aka local)